pkgs/top-level/python-packages.nix

```diff
@@ -7258,7 +7258,7 @@ in modules // {
 
     meta = with stdenv.lib; {
       description = "Python package to load images as numpy arrays";
-      homepage = https://readthedocs.org/projects/imread/;
+      homepage = http://imread.readthedocs.io/en/latest/;
```

Comment:
Good catch. I was updating to the readthedocs.io domain and copied and pasted an HTTP version.